cypresse2e/cypress/integration/e2eTests/helpBoard/helpBoard.spec.js

```diff
@@ -0,0 +1,47 @@
+import HelpBoard from '../../../elements/pages/helpBoard'
+
+describe('FightPandemics Help Board Page for unauthorized user', () => {
+
+    const helpBoard = new HelpBoard();
+    var h1Heading = "Help Board";
+    var noPostsAvailableText = "Sorry, there are currently no relevant posts available. Please try using a different filter search or";
+
+    context('User opens Help Board', () => {
+        beforeEach(() => {
+            helpBoard.visit();
+        });
+
+
+        it('FP logo is visible and clickable', () => {
+            cy.checkFpLogoIsVisibleAndClickable(helpBoard.getFpLogoLocator());
+
+        });
+        
+        it('Help Board page contains heading', () => {          
+            cy.pageContainsHeading(helpBoard.getHelpBoardPageTitleLocator(),h1Heading);
+        });
+
+        it('Unauthorized user is redirected to SignIn page when clicking Add Post Button', () => {          
+            var addPostButton = helpBoard.getAddPostButton();
```

Comment:
I see this practice here that you create a var while you could just write a single line of code instead like this:  
`var addPostButton = helpBoard.getAddPostButton();
            addPostButton.click({ force: true });`
` helpBoard.getAddPostButton().click({ force: true });`, but I saw the same practice in command.js files as well, but it's not necessary. What do you think about this?